WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -53,32 +52,55 @@ public static void Main(string[] args)
 				AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
 				TaskScheduler.UnobservedTaskException += TaskScheduler_UnobservedTaskException;
 
-				SingleInstanceChecker.EnsureSingleOrThrowAsync().GetAwaiter().GetResult();
-
-				runGui = ProcessCliCommands(args);
-
-				if (CrashReporter.IsReport)
+				if (crashReporter.TryProcessCliArgs(args))
```

Comment:
Should this be here?

Reply:
Yes, that is OK. Later ProcessCliCommands will be removed.